llpc/test/shaderdb/relocatable_shaders/PipelineVsFs_GlueCache.pipe

```diff
@@ -118,6 +118,6 @@ binding[0].binding = 1
 binding[0].stride = 16
 binding[0].inputRate = VK_VERTEX_INPUT_RATE_VERTEX
 attribute[0].location = 0
-attribute[0].binding = 0
+attribute[0].binding = 1
```

Comment:
This change seems to cause a test failure

Reply:
All tests pass now, thanks